contributing/lts-release.md

```diff
@@ -0,0 +1,29 @@
+# Long-Term Stable (LTS) Release Channel
+
+The **lts** release channel provides a previous **stable** build for four-week intervals. Every four weeks, the current **stable** release is promoted to **lts**. This is not recommended for all AMP publishers, but is provided so publishers performing a QA cycle may do so less often. Individual pages can explicitly opt into **lts**.
+
+> Note: The LTS release channel is available only to AMP HTML pages; it is not available to AMP Actions, AMP for Email, AMP Stories, or AMP for Ads.
+
+## How to Opt-In
+
+```html
+<!-- Standard AMP HTML runtime and extension scripts -->
+<script async src="https://cdn.ampproject.org/v0.js"></script>
+<script
+  async
+  custom-element="amp-ad"
+  src="https://cdn.ampproject.org/v0/amp-ad-0.1.js"
+></script>
+
+<!-- LTS AMP HTML runtime and extension scripts -->
+<script async src="https://cdn.ampproject.org/lts/v0.js"></script>
+<script
+  async
+  custom-element="amp-ad"
+  src="https://cdn.ampproject.org/lts/v0/amp-ad-0.1.js"
+></script>
+```
+
+As long as all runtime and extension scripts in a page must use the same release version, requesting the LTS scripts is considered valid AMP and gets the same cache benefits as the **stable** release channel.
```

Comment:
This wording is a bit confusing to me. Maybe:
"While the same version for the runtime and extension scripts is a requirement, requesting lts is valid AMP. It will get the same cache benefits as the stable release channel. "

Also note that `LTS` is capital - we should stick to a standard casing. Seems most is lowercase.

Reply:
Hahaha, wow, thats a can of worms. What I think it should come down to is the distinction for the reader. If its important for them to understand the differences between these, we should use more explicit language to explain that - if it isn't important, it might be easier to read if we stick to something standard. WDYT?

Reply:
I'll just update all uses to **lts** then

Reply:
Donezo.